Xwt/Xwt/Toolkit.cs

```diff
@@ -224,15 +224,26 @@ internal static Toolkit Load (string fullTypeName, bool isGuest)
 		/// Load a toolkit of a specified type.
 		/// </summary>
 		/// <param name="type">The toolkit type.</param>
-		public static Toolkit Load (ToolkitType type)
+		public static Toolkit Load(ToolkitType type)
+		{
+			return Load(type, false);
```

Comment:
```suggestion
			return Load(type, true);
```
Shouldn't this be `true` by default?

Reply:
No...
false means it's not a guest, and therefore shouldn't do the full initialization

Reply:
Yes, it should be false for this. It means "Do the full initialization"

So when Gtk is initialised through gtk.init, then Xwt.Gtk is the guest, and so doesn't need to do the initialization again (this isn't really an issue because Gtk allows Gtk.Init to be called multiple times)

When Cocoa is initialised through NSApplication.Initialize then Xwt.Mac is the guest, so doesn't need to do initialization

But when Xwt.Mac is running inside Gtk, then it is not the guest and so it needs to still do the initialization